src/openforms/submissions/models/submission.py

```diff
@@ -284,6 +285,14 @@ class Submission(models.Model):
         ),
     )
 
+    finalised_registration_backend_key = models.CharField(
+        _("final registration backend key"),
+        max_length=50,  # FormRegistrationBackend.key.max_length
+        null=True,
```

Comment:
we don't normally make text-based fields nullable, as the empty string is the empty value